tests/two_d_array_ut/two_d_array_ut.c

```diff
@@ -0,0 +1,642 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+#include <stdlib.h>
```

Comment:
Pretty sure you want to include inttypes also.

Reply:
added